assembly/dependencies-apache-ignite-lgpl.xml

```diff
@@ -141,6 +141,7 @@
                 <exclude>org.gridgain:ignite-ml-spark-model-parser</exclude>
                 <exclude>org.gridgain:ignite-ml-tensorflow-model-parser</exclude>
                 <exclude>org.gridgain:ignite-ml-xgboost-model-parser</exclude>
+                <exclude>org.gridgain:ignite-management-console-agent</exclude>
```

Comment:
@akuznetsov-gridgain We should remove this in GG-29327

Reply:
I agree, but then the change should also be done in GG-29327.

Reply:
I've already prepared PR fro GG-29327
we can merge branches if needed